remote/rc.xml

```diff
@@ -750,7 +750,7 @@
 
   # end of the example
 -->
-  <application name="Capella">
+  <application type="normal" title="workspace - Capella*">
```

Comment:
Add the same rule for Papyrus and Eclipse. Even though it wasn't tracked before, we should keep it aligned and have the same behaviour between Papyrus, Eclipse & Capella.